src/components/NewbieSelect/__tests__/NewbieSelect.tsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
```

Comment:
Could you change the name of the file to `NewbieSelect.test.tsx` to keep consistency?

Reply:
Done. This file wasn't even included in tests. 👍 
Fixed here 6968d4e7e8338a839706f0f60446463ecc90d6db and df0c5f9ab825a6913ccc7a4e0e1e2f2d2dc7f045